portal/src/app/components/home/components/experiment/components/table-options/TableOptions.tsx

```diff
@@ -0,0 +1,34 @@
+import styles from './TableOptions.module.scss';
+import { TABLE_OPTIONS_EN } from './translate/en';
+import SelectColumnsSvg from '../../../../../../../assets/images/select-columns.svg';
+import SelectColumnsCheckedSvg from '../../../../../../../assets/images/select-columns-checked.svg';
+import { Button, ButtonActionType, ButtonSize, ButtonStyleType } from '../../../../../../shared/components/att-button';
+
+const SelectColumnsAriaLabel: string = TABLE_OPTIONS_EN.SELECT_COLUMNS;
+
+export interface TableOptionsProps {
+  handleSelectColumnsClick: () => void;
+  isPopupOpen: boolean;
+  // data: AttSelectOption[];
+}
+
+export const TableOptions: React.FC<TableOptionsProps> = (props: TableOptionsProps) => {
+  return (
+    <div className={styles.table_options_wrapper}>
+      <Button
+        className={styles.close_button}
+        ariaLabel={SelectColumnsAriaLabel}
+        size={ButtonSize.NONE}
+        styleType={ButtonStyleType.WRAPPER}
+        actionType={ButtonActionType.BUTTON}
+        onButtonClick={props.handleSelectColumnsClick}
+      >
+        <div className={styles.option_wrapper}>
+          <img className={`${styles.option_image} ${styles.default_image}`} src={SelectColumnsSvg} alt={SelectColumnsAriaLabel} />
```

Comment:
use `cn` instead class name as string will do the same thing, 
`className ={cn(styles.option_image, styles.default_image)}`

for all 2 line